icon_validator/rules/workflow_validators/workflow_title_validator.py

```diff
@@ -1,49 +1,81 @@
 from icon_validator.rules.validator import KomandPluginValidator
 from icon_validator.exceptions import ValidationException
 from icon_validator.rules.lists.lists import title_validation_list
+import os
+import json
 
 
 class WorkflowTitleValidator(KomandPluginValidator):
 
-    def validate(self, spec):
-        """
-        Checks that title is not blank.
-        Checks that title does not end with a period.
-        Checks that title does not start with a lower case letter.
-        Checks that title does not start with a space.
-        Checks that title is 6 words or less.
-        Checks that title is properly capitalized.
-        """
-        if "title" not in spec.spec_dictionary():
-            raise ValidationException("Workflow title is missing.")
-
-        title = spec.spec_dictionary()["title"]
-
+    @staticmethod
+    def validate_title(title, file_type):
         if not isinstance(title, str):
-            raise ValidationException("Title must not be blank")
+            raise ValidationException(f"Title must not be blank in {file_type} file.")
         if title == "":
-            raise ValidationException("Title must not be blank")
+            raise ValidationException(f"Title must not be blank in {file_type} file.")
         if title.endswith("."):
-            raise ValidationException("Title ends with period when it should not.")
+            raise ValidationException(f"Title ends with period when it should not in {file_type} file.")
         if title[0].islower():
             # This plugin title is OK: minFraud
```

Comment:
Wouldn't minFraud and ifconfig.co trip the below exception?

Reply:
This would trip that.

Reply:
@blaxminarayan-r7 this comment was waiting responding and is the reason we did not merge this.

Reply:
@pmara-r7 We only add here more info to the Exception so when it raise we know exactly in which file the error occur. Could you please provide more info what you mean about `minFraud` and `ifconfig.co`?

Reply:
We were confused about the comment in the code (I see that you are not changing that) but we were just concerned about what that line actually does. For this release, there is no need to focus closely on this if there is no bug raised around it in my opinion.